src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -209,4 +204,15 @@ private function normalizeDomain($domain)
 
         return $normalizedDomain;
     }
+
+    private function replaceHtmlSpecialChars(string $message, string $functionName)
```

Comment:
Missing return type and phpdoc: https://www.pabloborowicz.com/why-you-should-be-commenting-your-code/#describing-class-members
```suggestion
    private function replaceHtmlSpecialChars(string $message, string $functionName):string
```